client/app/scripts/actions/app-actions.js

```diff
@@ -76,12 +78,20 @@ export function clickNode(nodeId, label, origin) {
     nodeId
   });
   updateRoute();
+  resetDetailsUpdateBuffer();
+  resumeUpdate();
   getNodeDetails(
```

Comment:
Instead of making sure we unpause in all these diff places it would be nice to have the abstraction that if we need to request fresh data to show some piece of info we have to unpause. I'm not sure where that would go, perhaps just something to keep in mind for now if we re-work the datamodel into something reduxey.